sql/core/src/main/java/org/apache/spark/sql/execution/streaming/StateMessage.proto

```diff
@@ -119,6 +136,35 @@ message AppendValue {
 message AppendList {
 }
 
+message GetValue {
+  bytes key = 1;
```

Comment:
Same as above, can we rename `key` as `userKey` to distinguish from grouping key?

Reply:
Updated